docs/extensions.md

```diff
@@ -185,4 +185,10 @@ To get results from geocoding data for providers who aren't using Open Data when
 
 ## Unlocking regions
 
+### Rio de Janeiro (Brazil) 🌎
+
+1. Go to [Fetransport Site](https://www.fetranspor.com.br/) and make sure you agree with the data terms. Contact us if you need help for this.
+2. Forward your confirmation mail to [api@skedgo.com](mailto:api@skedgo.com) 
+3. We will then unlock the region for your API key. 
+
 To get results for regions with providers that aren't using Open Data, or which we haven't yet been connected to our system, please get in touch with our team [by mail](mailto:api@skedgo.com) or on [Slack](http://slack.tripgo.com/) (by [self-invite](http://slackin.tripgo.com/)).
```

Comment:
Please move this old block before the new block on Rio. Otherwise it looks like this only applies to Rio.

Reply:
Done!